public/sass/pages/_dashboard.scss

```diff
@@ -102,11 +102,22 @@ div.flot-text {
   width: 100%;
   flex-grow: 1;
   height: calc(100% - #{$panel-header-height});
+  overflow: hidden;
```

Comment:
Note: this feels a bit dangerous. Do you know about any use cases that might introduce a bug by adding this code? @torkelo

Reply:
it breaks the dropdown select in Table panel, if we had a proper portal for Select component that could solve that

Reply:
hmm, damn...so I need to rethink this a bit.

Reply:
Let me know if you think this is something we should proceed with. I will wait with fixing the tests until I got a 👍🏻

Reply:
And I guess we want to support horizontal scrolling as well.